### DISCUSSION TITLE
Google Sign In popup  redirects to not page found

### DISCUSSION THREAD

**Discussion by ArgekarStudios** [hannoeru/vite-plugin-pages#121] [Q&A] [answered]
Hello, Developers I created a component that triggers GoogleSignInWithPopup.

When the popup opens it doesn't go to google sign in instead it redirects to the 404 page not found page.
When inspected error shows as same site cookie error.

Use vite and optionable template (https://github.com/antfu/vitesse)

const loginWithGoogle = () => {
  const provider = new GoogleAuthProvider()
  const auth = getAuth()
  signInWithPopup(auth, provider).then((res) => {}).catch((e) => {})

**Comment by ArgekarStudios** [comment]
any sloutions

**Comment by ArgekarStudios** [comment]
Here is a video link that explains the issue.
https://drive.google.com/file/d/19z41vgCa30Q-JjJinBkEAWqoORj8je7Y/view?usp=sharing
and this is repo link devlop branch
https://github.com/ArgekarStudios/digi_resume_web_issue

**Comment by hannoeru** [comment]
Hint: callback url setting.

**Comment by ArgekarStudios** [comment, marked as answer]
I did some tweaks.
instead of using somewebsite.com
I used www.somewebsite.com in popup redirect and it worked
